adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala

```diff
@@ -49,7 +49,7 @@ object ADAMContext {
   implicit def sparkContextToADAMContext(sc: SparkContext): ADAMContext = new ADAMContext(sc)
 
   // Add generic RDD methods for all types of ADAM RDDs
-  implicit def rddToADAMRDD[T <% SpecificRecord: Manifest](rdd: RDD[T]) = new ADAMRDDFunctions(rdd)
+  implicit def rddToADAMRDD[T](rdd: RDD[T])(implicit ev1: T => SpecificRecord, ev2: Manifest[T]): ADAMRDDFunctions[T] = new ADAMRDDFunctions(rdd)
```

Comment:
@laserson sure!  He is my attempt to explain it.

View Bounds, for which the syntax is as follows with <% :
def myFun[A <% B](a: A) : B = a
basically mean that A can be treated like B. These are deprecated in 2.10, here is the original thread:
https://groups.google.com/forum/#!topic/scala-internals/hNow9MvAi6Q/discussion.
Instead we require that A must have an implicit method which converts A to B with the new syntax:
def myFun[A](a: A)(implicit ev1: A => B): B = a
ev1 here is short for evidence 1.
Here is a good article on it:
http://jatinpuri.com/2014/03/replace-view-bounds/
I think the newer syntax is clearer, but thats really the difference to us mere mortals, syntax.